api/client/client.go

```diff
@@ -3389,3 +3389,66 @@ func (c *Client) GetHeadlessAuthentication(ctx context.Context, id string) (*typ
 	}
 	return headlessAuthn, nil
 }
+
+// CreatePlugins creates a new plugin resource
+func (c *Client) CreatePlugin(ctx context.Context, plugin types.Plugin) error {
+	p, ok := plugin.(*types.PluginV1)
+	if !ok {
+		return trace.BadParameter("unknown plugin type")
+	}
+	_, err := c.grpc.CreatePlugin(ctx, p, c.callOpts...)
```

Comment:
I think we are no longer passing in the options
If you rebase your branch, you should not see any usage of it